README.rst

```diff
@@ -85,6 +85,32 @@ directory you can do this::
 If there's not output. It worked. Check how it edited your
 requirements files.
 
+Filtering requirements by Python version
+========================================
+
+Some requirements have many wheels built for different versions of Python and
+different architectures. These hashes aren't useful in some cases, if those
+wheels don't work with your project. Hashin can filter on the Python version
+to eliminate these extraneous hashes.
```

Comment:
s/elimitate/skip

Eliminate sounds like a verb you do to something that already existed.